babel/core.py

```diff
@@ -1112,26 +1150,32 @@ def negotiate_locale(preferred: Iterable[str], available: Iterable[str], sep: st
     return None
 
 
-def parse_locale(identifier: str, sep: str = '_') -> tuple[str, str | None, str | None, str | None]:
+def parse_locale(identifier: str, sep: str = '_') \
+        -> tuple[str, str | None, str | None, str | None, str | None]:
```

Comment:
```suggestion
def parse_locale(
    identifier: str,
    sep: str = '_',
) -> tuple[str, str | None, str | None, str | None, str | None]:
```

Reply:
No discussion. I was running `black` on the code, and it would have done more or less the same, but it also changed 350 other lines, so I hit 'undo' ;)